zinc/src/sbt-test/source-dependencies/trait-trait-212/mirtest/Hello.scala

```diff
@@ -0,0 +1,8 @@
+package xx
+
+object Hello extends App {
+  val consumer = new StringLibSpecs
```

Comment:
This can be minimized further by removing `StringLibSpecs`:

```scala
package xx

object Hello extends App {
  (new EvaluatorSpecification).transform
}
```

Reply:
My earlier reproduction had that too, but because `new trait {}` captures the enclosing scope from `Hello`, I think it reduces the number of feature interaction if I used an actual class there.

```scala
Caused by: java.lang.AbstractMethodError: xx.StringLibSpecs.buildNonemptyObjects(II)V
```

as opposed to

```
Caused by: java.lang.AbstractMethodError: xx.Hello$$anon$1.buildNonemptyObjects(II)V
```